bgpd/bgp_vty.c

```diff
@@ -18300,6 +18304,31 @@ DEFPY(mpls_bgp_forwarding, mpls_bgp_forwarding_cmd,
 	return CMD_SUCCESS;
 }
 
+DEFPY (bgp_inq_limit,
+       bgp_inq_limit_cmd,
+       "bgp inq-limit (0-4294967295)$limit",
```

Comment:
Do we want to install a lower bounds for this?  I don't think it makes sense for a user to configure a limit of 0

Reply:
i like being able to test it. I can make it 1

Reply:
done, made it 1